### ISSUE TITLE
Sort dashboard exactly same as website

### ISSUE THREAD

**Issue by dreeves** [beeminder/BeeSwift#49]
```[tasklist]
### Desiderata
- [ ] Make it be the case that the smartphone apps can count on the API returning the goals in standard order
- [ ] TBD
- [ ] Update the [forum thread](https://forum.beeminder.com/t/order-differs-on-ios-app-vs-website/6690)
```

See [beebody gissue](https://github.com/beeminder/beeminder/issues/1390) and [forum thread](https://forum.beeminder.com/t/order-differs-on-ios-app-vs-website/6690).

Relatedly, I strongly dislike how the sort options are permanent in BeemiOS. On the website you can click a column header to re-sort but on next page load it will be back to sort-by-urgency. I think it's really bad to let beemergencies go out of sight. If the sort choice could be non-persistent, that'd be fine. If that's not so easy (or would be broken-seeming), I'm a little tempted to say we should kill the sorting option altogether.

See also https://forum.beeminder.com/t/my-beeminder-browser-extension/6648/23

## Cognata

* https://github.com/beeminder/beeminder/issues/1390
* https://forum.beeminder.com/t/order-differs-on-ios-app-vs-website/6690
* https://forum.beeminder.com/t/my-beeminder-browser-extension/6648/23

_**Verbata**: sort order, standard dashboard order, sorting by urgency,_

**Comment by krugerk** [comment]
Could it be as easy as using the list of goals provided under the user endpoint as the basis for the sort?

**Comment by dreeves** [comment]
(krugerk correctly asks if we could just use the ordering of goals as returned by the API)

**Comment by theospears** [comment]
Fixed by #451
